tools/pythonpkg/duckdb/experimental/spark/errors/__init__.py

```diff
@@ -64,6 +64,7 @@
     "SparkUpgradeException",
     "PySparkTypeError",
     "PySparkValueError",
+    "PySparkIndexError",
```

Comment:
I had to add this so that `PySparkIndexError` is importable with `from duckdb.experimental.spark.errors import *`. I assume it was not intended to exclude it from `__all__` but leaving this comment just in case there was a reason.

Reply:
No reason, likely just an oversight by the contributor responsible for adding it